docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '3'
+
+services:
+  cassandra:
+    build:
+      context: ./test/docker
+      dockerfile: Dockerfile.cassandra
```

Comment:
We're using a custom Dockerfile so we can change the config if `AUTHENTICATION` is `true`.